frontend/src/stimulus/controllers/dynamic/work-packages/activities-tab/index.controller.ts

```diff
@@ -556,6 +561,7 @@ export default class IndexController extends Controller {
     const editorElement = this.getCkEditorElement();
     if (editorElement) {
       editorElement.addEventListener('saveRequested', this.onSubmitBound);
+      editorElement.addEventListener('editorEscape', this.onEscapeEditorBound);
```

Comment:
We're likely missing a remove event listener.

Reply:
Ah, yeah- good eye! 👍🏾